DataCollector/PrettyDataCollector.php

```diff
@@ -84,7 +84,9 @@ public function collect(Request $request, Response $response, \Exception $except
                 }
 
                 // format the method call
-                if (isset($log['authenticate'])) {
+                if (isset($log['aggregate'])) {
+                    $query .= '.aggregate('.$this->bsonEncode($log['pipeline']).', '.$this->bsonEncode($log['options']).')';
```

Comment:
I think `options` should be included only when not empty to avoid having `[  ]` in logged query